scipy/stats/tests/test_multivariate.py

```diff
@@ -1429,6 +1429,315 @@ def test_samples(self):
         assert_allclose(actual, expected)
 
 
+class TestMatrixT:
+
+    def test_bad_input(self):
+        # Check that bad inputs raise errors
+        num_rows = 4
+        num_cols = 3
+        df = 5
+        M = np.full((num_rows,num_cols), 0.3)
+        U = 0.5 * np.identity(num_rows) + np.full((num_rows, num_rows), 0.5)
+        V = 0.7 * np.identity(num_cols) + np.full((num_cols, num_cols), 0.3)
+
+        # Nonpositive degrees of freedom
+        assert_raises(ValueError, matrix_t, 0, 1, 1, 0)
+        assert_raises(ValueError, matrix_t, 0, 1, 1, -1)
```

Comment:
These and all following `assert_raises` need to check the error message using the `match=` clause from pytest.